runtime/caml/domain_state.h

```diff
@@ -45,23 +43,29 @@ enum {
 #undef DOMAIN_STATE
 };
 
-#ifdef CAML_NAME_SPACE
+/* TODO: CAML_NAME_SPACE working? */
```

Comment:
Either clarify this comment or remove it.

Reply:
I'll remove this comment assuming that `CAML_NAME_SPACE` is defined. See comment in https://github.com/ocaml/ocaml/pull/10831/files#r776684663.